roles/common/README.md

```diff
@@ -0,0 +1,77 @@
+common
+======
+
+The tests in this role are not specific to any one functional area but are an
+aggregate of common tests that can be used in all OSP 18.0/OCP jobs.
+
+The available tasks tests are:
+
+* pod tests
```

Comment:
Please update this to include the added tests.

The whole review will get easier once the pod tests merge, which I am trying to do ASAP.